packages/builder/src/components/userInterface/propertyCategories.js

```diff
@@ -167,16 +167,15 @@ export const typography = [
     key: "font-weight",
     control: OptionSelect,
     options: [
-      { label: "100", value:"100"},
-      { label: "200", value:"200"},
-      { label: "300", value:"300"},
-      { label: "400", value:"400"},
-      { label: "500", value:"500"},
-      { label: "600", value:"600"},
-      { label: "700", value:"700"},
-      { label: "800", value:"800"},
-      { label: "900", value:"900"},
-
+      { label: "100", value: "100" },
```

Comment:
Nice work mate - just to let you know. If the label and the value for each options are the same, you can save yourself some bother and give it an array of strings....
`options: ["100", "200", "300", "400","500"]`